Either/compactable

```diff
@@ -0,0 +1,31 @@
+{-
+mapOptional : (a -> Optional b) -> Either m a -> Either m b
```

Comment:
I don’t think this is useful documentation, especially since this is a type class instance.

Reply:
Thanks for catching, I actually just put it there to help me implement it :sweat_smile: